src/Microsoft.DotNet.Darc/src/Darc/Helpers/LocalSettings.cs

```diff
@@ -50,12 +52,29 @@ public static LocalSettings LoadSettingsFile()
         /// <remarks>The command line takes precedence over the darc settings file.</remarks>
         public static DarcSettings GetDarcSettings(CommandLineOptions options, ILogger logger, string repoUri = null)
         {
+            LocalSettings localSettings = null;
             DarcSettings darcSettings = new DarcSettings();
             darcSettings.GitType = GitRepoType.None;
 
             try
             {
-                LocalSettings localSettings = LoadSettingsFile();
+                try
+                {
+                    localSettings = LoadSettingsFile();
+                }
+                catch (Exception exc) when (exc is DirectoryNotFoundException || exc is FileNotFoundException)
+                {
+                    if (string.IsNullOrEmpty(options.AzureDevOpsPat) &&
+                        string.IsNullOrEmpty(options.GitHubPat) &&
+                        string.IsNullOrEmpty(options.BuildAssetRegistryPassword))
+                    {
+                        throw new DarcException("Please make sure to run darc authenticate and set" +
+                            " 'bar_password' and 'github_token' or 'azure_devops_token' or append" +
+                            "'-p <bar_password>' [--github-pat <githug_token> | " +
+                            "--azdev-pat <azure_devops_token> to your command");
```

Comment:
Also missing closing `]`
```suggestion
                            "--azdev-pat <azure_devops_token>] to your command");
```